_homework/web-development/portfolio-creation.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Create your own webpage
+assigned: Oct 1, 2018
+due: Oct 2, 2018
+---
+
+
+Tasks
+--------
+
+- Fork Rapha's [repository](https://github.com/raphamontenegro)
```

Comment:
At first glance, it doesn't seem like this link is a repo, just your main page. Should this link somewhere else?

Reply:
I'll create this repository tomorrow in class with them with the quick demo.  For now it is just a placeholder.

Reply:
OK cool 👍